### DISCUSSION TITLE
Animals on Rescueland

### DISCUSSION THREAD

**Discussion by Klesone** [hermonochy/Rescueland#20] [Q&A] [answered]
Are there any animals on Rescueland and if so which ones?

**Comment by hermonochy** [comment, marked as answer]
Yes! Animals do exist on Rescueland, with millions of varieties. To name a few:

- Zypha are large snake-like beings bioluminescent skin that emits a spectrum of colours. They are known for their ability to camouflage themselves almost anywhere.
- Etecha are flying beings with wingspans reaching up to 88 metres. They have feathered bodies that are adapted to high altitudes and are equipped with specialized air sacs for efficient flight.

**Comment by Vlada3333** [comment]
Thanks for the information!
